lib/facter/self_service.rb

```diff
@@ -108,6 +108,12 @@
     { S0013: File.open(summary_path).read.include?('catalog_application') }
   end
 
+  chunk(:S0014) do
+    file_time = File.mtime('/opt/puppetlabs/server/data/puppetdb/stockpile/cmd/q')
```

Comment:
Wont this show the last time the folder was modified, so this would be the timestamp of the newest file in the folder? and not the timestamp of the oldest file in the folder?